### PR TITLE
Following Changes are made based on issues:

### PR DESCRIPTION
**Pull request by hemant2705**
Issue 1: No Blank Action bar theme was there
changes made: A theme named Algophantoms1 was made having ho action bar in it.(to use it for splash screen and various activities)
Issue 2:Splash screen was blank #11
changes made:A modern splash screen ui added with a loader with help of lottie files and a handler which is transferring the control from splashscreen to login screen.
Issue 3:On back Pressed #17
(while pressing back button app flow should be exited but right now its sending us just back)
changes made: a function onbackpressed is added to login activity and MainActivity.kt

### CODE REVIEW

**Comment by dheerajkotwani** [comment]
@hemant2705 kindly resolve the conflicts and add the screenshots for the changes.

**Comment by hemant2705** [comment]
Can you check it now please

File: app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -13,4 +13,17 @@
         <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimary</item>
         <!-- Customize your theme here. -->
     </style>
+    <style name="Theme.AlgoPhantoms1" parent="Theme.MaterialComponents.DayNight.NoActionBar">
```
Comment (carotkut94) [inline app/src/main/res/values/themes.xml:16]: consider naming themes based on what it holds, for there `Theme.AlgoPahantoms.NoActionBar` makes more sense

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 2]
```diff
@@ -4,6 +4,54 @@
...
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toEndOf="@id/icon"
+            app:layout_constraintTop_toTopOf="parent" />
+
+
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <com.airbnb.lottie.LottieAnimationView
+        android:layout_width="200dp"
```
Comment (carotkut94) [inline app/src/main/res/layout/activity_splash_screen.xml:45]: always keep the dimens values in `dimens.xml`

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 3]
```diff
@@ -4,6 +4,54 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+ android:background="@color/black"
     tools:context=".activities.SplashScreenActivity">
 
+    <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (carotkut94) [inline app/src/main/res/layout/activity_splash_screen.xml:10]: this unnecessary nesting of the constraint layout can be avoided

**Review by humbledroid — COMMENTED** [review]
minor nit picks

File: app/src/main/res/values/themes.xml (hunk 1 above)
Comment (hemant2705) [reply app/src/main/res/values/themes.xml:16]: yeah . Thanks for pointing out. let me commit required changes and let you know.

**Comment by hemant2705** [comment]
can you please check it now !

**Comment by Shreyash41** [comment]
@hemant2705 #11 This issue is assigned to me. Why are you make changes to it I am already working on it. Please work on your issues that are assigned to you. Please see this concern @carotkut94 @dheerajkotwani

**Comment by hemant2705** [comment]
@Shreyash41 True , the issue had been assigned to you, but please see the date when I made this pr.
its 12 days back when the project was hardly maintained by anyone hence I marked down the whatever possible changes I could contribute and dropped the pr. It wasn't intentional to work upon a issue which has been assigned to you. 
Maintainers please this concern.

**Comment by dheerajkotwani** [comment]
> 
> 
> can you please check it now !

You didn't provided the Screenshots related to Splash Screen also try solving single issue in one PR from next time.

**Pull request closed without merging**